x-pack/plugins/enterprise_search/public/applications/shared/add_content_empty_prompt/add_content_empty_prompt.tsx

```diff
@@ -15,99 +15,95 @@ import {
-                      defaultMessage: 'Learn more',
-                    })}
-                  </EuiLink>
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiImage
-            size="xl"
-            float="right"
-            src={colorMode === 'LIGHT' ? welcomeGraphicLight : welcomeGraphicDark}
-            alt={i18n.translate('xpack.enterpriseSearch.overview.searchIndices.image.altText', {
-              defaultMessage: 'Search indices illustration',
-            })}
+    <EuiEmptyPrompt
+      layout="horizontal"
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.enterpriseSearch.addContentEmptyPrompt.h2.createYourFirstIndexLabel"
+            defaultMessage="Create your first Index"
           />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </EuiPanel>
+        </h2>
+      }
+      body={
+        <EuiFlexGroup direction="column">
+          <EuiFlexItem>
+            {i18n.translate(
+              'xpack.enterpriseSearch.addContentEmptyPrompt.onceCreatedThisPageLabel',
+              {
+                defaultMessage: 'Once created, this page will list all of your indices',
+              }
+            )}
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <div>
+              <EuiLinkTo
+                to={generatePath(ENTERPRISE_SEARCH_CONTENT_PLUGIN.URL + NEW_INDEX_PATH)}
+                shouldNotCreateHref
+              >
+                <EuiButton color="primary" fill iconType="plusInCircle">
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.addContentEmptyPrompt.newIndexButtonLabel',
+                    {
+                      defaultMessage: 'New Index',
+                    }
+                  )}
+                </EuiButton>
+              </EuiLinkTo>
+            </div>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      }
+      footer={
+        <EuiFlexGroup gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <b>
+              {i18n.translate('xpack.enterpriseSearch.addContentEmptyPrompt.wantToLearnMoreLabel', {
+                defaultMessage: 'Want to learn more?',
+              })}
+            </b>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiLink href={docLinks.start} target="_blank">
+              {i18n.translate('xpack.enterpriseSearch.overview.emptyState.footerLinkTitle', {
+                defaultMessage: 'Learn more',
+              })}
+            </EuiLink>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      }
+      hasBorder
+      icon={
+        <EuiImage
+          size="fullWidth"
+          src={colorMode === 'LIGHT' ? welcomeGraphicLight : welcomeGraphicDark}
+          alt={i18n.translate('xpack.enterpriseSearch.overview.searchIndices.image.altText', {
```

Comment:
I don't think this needs an alt as it's purely decorative, doesn't convey any information

Reply:
it will be skipped when empty right

Reply:
Yeah, you should set it to an empty string for images that don't convey any information/are purely decorative.